app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.makeiteven2.fragments
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.makeiteven2.R
+import com.example.makeiteven2.adapters.LevelsAdapter
+import com.example.makeiteven2.data_models.Level
+import kotlinx.android.synthetic.main.fragment_levels.view.*
+
+class FragmentLevelsScreen : Fragment() {
+
+    private lateinit var callBack : OnLevelClicked
+    private lateinit var mLevelsRecyclerView : RecyclerView
+    private lateinit var mRecyclerViewAdapter : LevelsAdapter
```

Comment:
consider to change name of "mRecyclerViewAdapter"  to "mLevelsAdapter"  more reasonable

Reply:
done